src/PlusDataCollection/Andor/vtkPlusAndorVideoSource.cxx

```diff
@@ -532,7 +531,7 @@ PlusStatus vtkPlusAndorVideoSource::AcquireFrame()
 {
   checkStatus(::SetExposureTime(this->effectiveExpTime), "SetExposureTime");
   checkStatus(::SetShutter(1, this->effectiveShutter, 0, 0), "SetShutter");
-  checkStatus(::SetImage(this->effectiveHBins, this->effectiveVBins, 1, 1024, 1, 1024), "Binning");
+  checkStatus(::SetImage(this->effectiveHBins, this->effectiveVBins, 1, frameSize[0], 1, frameSize[1]), "Binning");
```

Comment:
I don't believe this is enforcing `frameSize[0]` and `frameSize[1]` to be the detector size.  It could be the frame size from the previous adjust buffers call. So if sensor is 2048x2048 a binning of 2 would produce a 1024x1024 frame and then the next capture would use 1024x1024 values.

`SetImage` is done like the following for `SetVerticalBins`:
```c++
int x, y;
checkStatus(GetDetector(&x, &y), "GetDetector");  // full sensor size
unsigned status = checkStatus(::SetImage(bins, this->VerticalBins, 1, x, 1, y), "SetImage");
```

In `InitializeAndorCamera` it is like:
```c++
  int x, y;
  checkStatus(GetDetector(&x, &y), "GetDetector");
  frameSize[0] = static_cast<unsigned>(x);
  frameSize[1] = static_cast<unsigned>(y);
```